src/test/municipality.test.ts

```diff
@@ -0,0 +1,76 @@
+import {
```

Comment:
by convention tests should be under `__tests__`, not `test`

Reply:
you also want to add a circleci config to run the tests now, see an example [here](https://github.com/teamdigitale/io-template-typescript/tree/master/.circleci)

Reply:
Done!